### ISSUE TITLE
Show full hierarchy in search and widget

### ISSUE THREAD

**Issue by Sirove** [orgzly/orgzly-android#351]
Maybe I missed the configuration but if not, it would be nice to have an option to display the full hierarchy of a TODO which is part of a bigger task. This way it'd be easy to keep the bigger picture in mind.

**Comment by nevenz** [comment]
Yes, this is not defined. I assume listing tags from higher-level notes first would make most sense?

**Comment by Sirove** [comment]
I'd say it's the other way around because of sorting tags like `b book `.

**Comment by Sirove** [comment]
I'd say it's the other way around so you can see the specifics on the widget even if the name of the task gets too long.
